src/transformers/models/time_series_transformer/modeling_time_series_transformer.py

```diff
@@ -1867,19 +1962,20 @@ def forward(
             encoder_last_hidden_state=outputs.encoder_last_hidden_state,
             encoder_hidden_states=outputs.encoder_hidden_states,
             encoder_attentions=outputs.encoder_attentions,
+            loc=outputs.loc,
             scale=outputs.scale,
             static_features=outputs.static_features,
         )
 
     @torch.no_grad()
     def generate(
         self,
-        static_categorical_features: torch.Tensor,
-        static_real_features: torch.Tensor,
         past_time_features: torch.Tensor,
         past_values: torch.Tensor,
         past_observed_mask: torch.Tensor,
```

Comment:
```suggestion
        past_values: torch.Tensor,
        past_time_features: torch.Tensor,
        past_observed_mask: torch.Tensor,
```
If we're breaking the order here, let's also use the same order as the forward methods